### DISCUSSION TITLE
时光相册 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#21] [模板讨论]
时光相册 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by wchen9** [comment]
登录直接报401， 尝试用过 https://github.com/ICE99125/everphoto_checkin/blob/main/everphoto.py 这个版本，但是报错API不存在，供参考。

**Reply by wjf0214** [reply]
> 登录直接报401， 尝试用过 [ICE99125/everphoto_checkin@`main`/everphoto.py](https://github.com/ICE99125/everphoto_checkin/blob/main/everphoto.py) 这个版本，但是报错API不存在，供参考。

我自己的正常，你这应该密码错误了，密码不是原密码。要抓登录请求的密码。以及账号有带86么。

**Reply by wchen9** [reply]
感谢，确实是的，我贴的那个里面的代码有密码加密转换所以可以登录。

**Comment by NewDeparture** [comment]
可以使用网站[https://md5jiami.bmcx.com/](https://md5jiami.bmcx.com/)的md5加密工具。加密内容为"tc.everphoto.password"，不包括引号，“password”替换为你的原密码，32位小写加密结果对应的就是加密之后的密码。
抓包方法参考[https://juejin.cn/post/7015035217773592583](https://juejin.cn/post/7015035217773592583)

**Reply by wjf0214** [reply]
更新了，只用填手机号和密码就行了。

**Reply by NewDeparture** [reply]
感谢更新
